### DISCUSSION TITLE
求激活码

### DISCUSSION THREAD

**Discussion by lsq6789** [LFTEC/LearningFucker#34] [General]
感谢
------------------------------License------------------------------
bKfXEqqyOrzggd+mUBjOV5OpDAVKf4Vc5vQJU7Nz1A/o7evGQ3/fE4TiDkrbnR849HX
KfbG6op98S+iBRdWs4OA3vBt7Dof1Kg0wADsojTLDSI3x755zvHBE0Uv+CCAmw9wogQ
o7w161ue70FUJ+V5rZNRTISbxV7L3v0kn09m/c2wkekF++Z0GVuZ9OFvqtjqTQOlvrM
TiNHwxjrEK4purwR7f352d40QTsTnCGzLrYvXKHVhyMXFAfhcQw9aCbo5O+Q2xQSLjn
PhCKP4ROh2PMTBDJIPfIn3EU0DpmyuldgKbn5s7ZdRwvT2o6M3lzjfJvgq1bJ9S4Yy3
sd4onXZK8I7Ef3Fs1oUW4EB+FEpi3v9YBXnY+CqNMSl5Yz/vPrRIJKSrzQXHq9MEAdm
/u+UYtUbviE/OSdegtZKD2iGwmSXmCOnuwUV0/be2Fb+i5w3y5kQlz341m0u4Wz8XPu
8irkiQ74Z3vt5cox8fPRK5N/lKXO3ZH6HA10IhK462v+31t+wXRgaLr3DHXC3tF6vM8
HnfF5hjRCDTNGNhxd9yZCFBDebAAk5fvxsd73TRzu+LbYkEiaHWqJen0941DWCSNxsi
kIqRUQmmHGg7APnFinTagNQE5w3IYhXUZT3aWLyr5u5JWgQ2k0JnH1laj7Q+gWWkfDd
sxGbnqOWC+Mi1G1+dsOeechC1mKVMMRQ7MqcWufXHxfTn1Nm3lEfleMbz6H+tal5Feg
HozTTYIJQpoT/IkW/yAGb/m+fQqI2WqNkTb8hlBRbhZei731uiIntyXhMRaAL13zHD4
2yUTgpQvCb8A6NqUUpgyrAsQr3JbjMCKLSjcDO8fDFvUM6Z/cOE4SmcvwSlb0cQtrxN
3Jyi3rxR1X87n/aKlmk03KUxr5n8GzCim6bXPKly+ZFMHU3pC36kJr4oZgsWW5F3QUo
/5bqxWZ87iJuWmQY2JYsk2EFbRtHlet5LUq3mj92SRSd9sPuHu8c2NSyQ1Qym/3VRO2
KswFJppLShvO0Vwypj0bGL5P4CFtqibAkbIAz+QRv9qcsPhdQAZx9t0p+eFShJL8SEM
hMLL4aSCMmYaQw3OpkdBP4o/NBmQkRvL1xhCknH6pvtqcF0+R8pRX6w7rHlwT/j0gTs
QpUljlTeGnOCuXuxtio4W7+cT8uQrR+jF/7B9SsyLMvMUVxaPBeXAaDJY9mgLyY4BHc
e6gj/enRIVFcZtutj/86qwnsNm9/LJ3HKn4PcLjLUZQiuyc+OcdTflf9oP/QYrpvU2D
P+YrZ9czSE+U22DZHLprvp+JYJ62bbV/tBK9HgQrax5GkW/kwjZkODlRx2P2PoUA9Tp
nAsZIuh8xTVH4jOdkux1YeYudw==
------------------------------License------------------------------

**Comment by LFTEC** [comment]
Mh93M3yMyqW+2pnUHzofW/TMZanB3R0RttmQTv9HTKuo45IZPcd/A6ZkNHgViqm1ZGrJAFbu4XJBFLWF4NU/kQrs1Y/g9DGwCC3hg8NzuL76T82ZIO2aV4zeW9F8WQdD6nOB5lcNB1D1ppJyRW5lREePVw6vCppXcRcW967s79H0rzf6xKzWS77X7ijjOMRbtHXdemWNwNBJ4x90kTwbzzAo+gCHYhJqp8HMI06qXofG1aY/iq6eSWjlWrq4oomqylDMSRRAP/6lFA5672l8uk+Qpj6t00bZp5R9ENi/tnd+BX9kntwsz+XkcS/QI91+JqcT91odwjmegpCd+MhHiWdOHkU1n4csu6jhq1qHF3SsmJVN04p5wJ7if4wARwr/Rta4PNUWlI6duf1rt+GpgGS/ROE3JKlSUfzJxDY0uD4F5uKUfjcGnPGtGW62jLB26O/h9LTJd8B2s/K2HiDQxql1c+OnLlABd5jpkBS5XrW6OXgRCm2GB/HIV61TsqUs6nxxw4dXodYc+ZRh4ReIYQ5J6gukb1+myEzTK3zWn24RCOpE2K3+v48bd6Y+my4WAunHuVfQEoeaRhnsRCf/DTO1SGwikESVZSWKTt6W1oFb2DYr32mORUz48iLPDDgg22BTW7msXWJbeCAAJg+jyc5L7WGNxuOzuE3F+qs4pmx2Su/T67o4JsesbDe46hXWFPOJzDsNhAVkEdOa9yl3nFeWHSrmVTh6QoK23t13yZS2RMvdvic/FuK8BiJP6a6fWHXnM/ynKMdn2r1hVOW6h/RKzd8oQmpamDmxIvBvIgOfxXKGIw8Emd4gFfZUXCmF0HBNKrE5eugsZSs/xtXuXdKeTTfwzZWjG8wbEVbOHgFnakYFABzQ2EpAWDVM5NxOXkjArG28ArkLSqJIEDqefo1vHEFlqCkAIxydhFEtI3u56OLD9SjqZYJSNCd4nluNzn+I+6MxE0yo8bD3DwNdG40IZjuFDk1L+fwPNhXYuAMCxaEv9Mmv/GwfeouWNORSQ2Rrl4bIDcjEkkhPf+/WPCWC0ssjcgwI+71M+JDJvQ/tpS6dQMGUmhqu425XzS6pGjVIcCVI+bTXYd1SUz37sQoKef0hStwFtdmC9DofLUfuSQuPX6ti21r3AfI9zi0Z6XrK/IkBr38lpq2HPocuvQkGmwTB5qktHWQM9bPjLU4Z4pJb2NFX4FxZu9jrtaIxviPToG6GJyIlBR1b85tB4D2/QQbct8wK5h6MEZX4FVksinaVH9oFOPByiBOA8C5SvBOBnosIsh0x8gThLKlOMvK0RQPsEibBvche1hhy/H9HfI2YC7FoNh1iw1ytvW4zW5DX6rcF8VsLMQX8rNuvEA==

**Reply by lsq6789** [reply]
大佬你好，我以前没有接触过github，只用过油猴的脚本。我跟着你的教程说明激活成功了，但启动方法我有些看不懂，包括下载对应平台的压缩包是什么压缩包，还有怎么将解压缩的目录指定为工作目录，大佬你能详细说一下吗，非常感谢

**Reply by LFTEC** [reply]
能生成激活码，说明你已经安装好了，后面就是按照命令去执行就好了，不用管压缩包和工作目录的事了
